result/yoda.js

```diff
@@ -0,0 +1,74 @@
+if ("red" === color) {
+    // ...
+}
+if (color === "red") {
+    // ...
+}
+/*eslint yoda: "error"*/
+
+if ("red" === color) {
+    // ...
+}
+
+if (true == flag) {
+    // ...
+}
+
+if (5 > count) {
+    // ...
+}
+
+if (-1 < str.indexOf(substr)) {
+    // ...
+}
+
+if (0 <= x && x < 1) {
+    // ...
+}
+/*eslint yoda: "error"*/
+
+if (5 & value) {
+    // ...
+}
+
+if (value === "red") {
+    // ...
+}
+/*eslint yoda: ["error", "never", { "exceptRange": true }]*/
+
+function isReddish(color) {
```

Comment:
__[development][eslint]__
* Error - 'isReddish' is defined but never used (no-unused-vars)